analysis/analyze_gor.py

```diff
@@ -28,24 +29,15 @@ def __init__(self, storage: Storage) -> None:
 
     def process_game(self, game: GameRecord) -> GorAnalytics:
         if game.black_manual_rank_update is not None:
-            self._storage.clear_set_count(game.black_id)
-            self._storage.set(game.black_id, GorEntry(rank_to_rating(game.black_manual_rank_update)))
+            storage.clear_set_count(game.black_id)
```

Comment:
Yup; fixed in 82a679a3c538c331663028edb43af412babdfb60. Happened because at some point I thought that should move into the helper, and I wasn't careful when moving it back.

Actually, I haven't tested these other analyze scripts at all really, just the one-at-a-time script. Still haven't; let me know if that makes you uncomfortable and I can check them out, but from the README it sounded like you weren't even sure they worked anymore.

Reply:
Nah one at a time is what we're using in production anyways.   Thanks for the quick fixes!